hudi-common/src/main/java/org/apache/hudi/common/fs/StorageSchemes.java

```diff
@@ -53,7 +53,9 @@
   // Databricks file system
   DBFS("dbfs", false),
   // IBM Cloud Object Storage
-  COS("cos", false);
+  COS("cos", false),
+  // Huawei Cloud Object Storage
+  OBS("obs", false);
```

Comment:
@xiarixiaoyao would you please also add an entry here and a page like this http://hudi.apache.org/docs/ibm_cos_hoodie.html to describe how to use obs?

Reply:
ok.  but maybe i have no right to edit  http://hudi.apache.org/docs/cloud.html, could you help me, thanks

Reply:
@xiarixiaoyao it is on asf-site branch and you would open a PR to fix.

Reply:
sorry for later， i will try today。 thanks @leesf